docker/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3'
 services:
   graph-node:
     image: graphprotocol/graph-node
-    ports:
+    expose:
```

Comment:
I would expect the whole point of running these is to expose the ports to the host. People should not be running production setups from this file. For this reason I do not think we should make this change, it would make it useless for development